src/AppBundle/Entity/CustomerRepository.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\EntityRepository;
+
+/**
+ * CustomerRepository
+ *
+ * This class was generated by the Doctrine ORM. Add your own custom
+ * repository methods below.
+ */
+class CustomerRepository extends EntityRepository
+{
+    public function findAllWithContactsAndOrders()
+    {
+        $query = $this->getEntityManager()->createQuery(
+            'SELECT customer, c, COUNT(o)
+            FROM AppBundle:Customer customer
+            JOIN customer.contacts c JOIN customer.orders o
+            GROUP BY customer.id, c.id'
+        );
+
+        return $query->getResult();
```

Comment:
to nie powinny być obiekty (getResult) tylko tablica - bo mimo że zapytanie wykonuje się w ~20ms, tak cała akcja (generowanie obiektów itp.) trwa 2 sekundy